internal/services/relayer/main.go

```diff
@@ -177,6 +192,8 @@ func (c *relayerConsumer) processTransfer(ctx context.Context, task data.RelayTa
 	switch {
 	case transfer.To.Chain == types.Near:
 		return c.processNearTransfer(task, transferDetails)
+	case transfer.To.Chain == types.Rarimo:
```

Comment:
It would be cleaner to implement the [Bridger](https://github.com/rarimo/relayer-svc/blob/e39299ed02a826ff9b52da831470fa2c92bb32fb/internal/services/bridger/main.go#L32) interface for Rarimo and Near.

This way, we can remove this switch and leave just:

```go
return c.bridgerProvider.GetBridger(transfer.To.Chain).Withdraw(ctx, transferDetails)
```